src/Offer/ReadModel/JSONLD/OfferLDProjector.php

```diff
@@ -418,7 +418,9 @@ protected function applyImageRemoved(AbstractImageRemoved $imageRemoved): ?JsonD
         );
 
         // Unset the main image if it matches the removed image
-        if (isset($offerLd->image) && $offerLd->{'image'} === $imageUrl) {
+        // stristr() comparison is done for edge cases where an offer has multiple images
+        // @see III-4684
```

Comment:
Maybe make this a real link?

Reply:
I've thought about this, but I had my doubts because It could crash the projector If there still some faulty/LegacyUrls in the eventStore. And it would still require some Url-manipulation. But maybe it would be easier to create a `sameAsProtocolAgnostic()` in `Url`?

Reply:
I was talking about the ticket number, but I admit that was not clear 😉